instructions-for-exchanges.md

```diff
@@ -0,0 +1,54 @@
+# Instructions for Exchanges
+
+This document contains `xud` setup instructions for exchange admins and technical users utilizing Docker.
+
+We use `docker-compose` to package `xud`, `mysql`, `lnd`, `btcd`, `ltcd`, `raiden` & `geth` together to make deploying these services as easy as typing a few commands. All configuration between `xud`, `mysql` and other containers are handled automatically by their `docker-compose` config file.
+
+The goal of this is `docker-compose` being able to lauch `xud` along with its dependencies.
+
+## 1. Prerequisites
+* docker (e.g. apt install docker-compose)
+* docker-compose (e.g. apt install docker-compose)
+
+To install these on e.g. Ubuntu, run:
```

Comment:
I would prefer if you link to the [Docker documentation](https://docs.docker.com/install/linux/docker-ce/ubuntu/) instead of writing the commands itself. Especially because the Docker packages in the Ubuntu repositories are fairly outdated and we should use the latest version.

Reply:
Agree with @michael1011

Reply:
Is the idea that exchanges should only be installing via docker? I wouldn't expect that to always be the case.

Reply:
Fixed.

@sangaman Both are not mutually exclusive, that's why I added the (manual) (docker) hints. It's just much faster to make a decision which installation method to use when adding a role.